modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -373,6 +373,19 @@ Extra Field Set:: Specifies extra fields.
 Registrations Require Approval:: Select whether registration requires approval.
 Approver ID:: Specifies the approver ID.
 
+== Spec Downloader
+
+The Spec Downloader component enables users to download an API specification.
+
+The API specification downloaded from API Community Manager is the same as the specification downloaded from Anypoint Exchange.
+
+The available buttons are *Download Spec as RAML* and *Download Spec as OAS*. If the API specification is uploaded as RAML and downloaded as OAS, or uploaded as OAS and downloaded as RAML, then the downloaded version is automatically generated and may have small inconsistencies with the uploaded version.
+
+You can specify values for the following parameters of the Spec Downloader component:
```

Comment:
So, after the user chooses the download type, the user customizes using parameters?
Or is this what the admin does?
How are these parameters related to the previous text about the buttons? Do we need a screenshot?
If these are UI labels, they should be formatted as such (* *).

Reply:
ACM uses Salesforce Community Cloud, which is an interactive web page for building interactive web pages. The admin uses ACM to drag and drop the components listed on this page into a page for developer users and then clicks "publish" to build it. So the admin will drag and drop a Spec Downloader onto the page, and specify the Record Id and Layout options, and the users will just see buttons to click: the *Download Spec as RAML* or *Download Spec as OAS* buttons. "Vertical" and "Horizontal" are options in a drop-down menu.